dhnx/optimization/precalc_hydraulic.py

```diff
@@ -25,48 +25,345 @@
 #   - https://www.schweizer-fn.de/stroemung/rauhigkeit/rauhigkeit.php
 
 
-def delta_p(v, d_i, k=0.1, T_medium=90, l=1,
-            pressure=1, fluid='IF97::Water'):
+def eq_smooth(x, *R_e):
+    r"""
+    Equation to be solved for smooth surfaces
```

Comment:
What does the function actually calculate? Is there a name of the functions (some functions are related to name/person (Prandtl, Blasius, ...)

```suggestion
    Equation for the calculation of the pressure drop of hydraulic smooth surfaces
```